src/processing_provider/provider.py

```diff
@@ -20,10 +20,16 @@
 """
 
 from qgis.core import QgsProcessingProvider
+from processing.core.ProcessingConfig import ProcessingConfig, Setting
+from processing.gui.ProviderActions import (ProviderContextMenuActions)
```

Comment:
```suggestion
from processing.gui.ProviderActions import ProviderContextMenuActions
```
Nothing wrong with that, just for consistency ;)